core/i18nilize/views.py

```diff
@@ -17,4 +19,33 @@ def post(self, request, *args, **kwargs):
             'message': 'Data received successfully!',
             'received_data': received_data
         }
-        return Response(response_data, status=status.HTTP_201_CREATED)
\ No newline at end of file
+        return Response(response_data, status=status.HTTP_201_CREATED)
+
+class CreateTokenView(APIView):
+    """
+    Endpoint to create a new token.
+    """
+    def post(self, request):
+        token = Token.objects.create()
+        data = {
+            'id': token.id,
+            'value': str(token.value),
+            'created_at': token.created_at.isoformat()
+        }
+        return Response(data, status=status.HTTP_201_CREATED)
+
+class ReadTokenView(APIView):
+    """
+    Endpoint to retrieve a token by its ID.
+    """
+    def get(self, request, pk):
+        try:
+            token = Token.objects.get(pk=pk)
+            data = {
+                'id': token.id,
+                'value': str(token.value),
+                'created_at': token.created_at.isoformat()
+            }
+            return Response(data, status=status.HTTP_200_OK)
+        except Token.DoesNotExist:
+            return Response({'error': 'Token not found.'}, status=status.HTTP_404_NOT_FOUND)
```

Comment:
wrap this into one APIVIew called TokenView, 
get and post should be functions underneath the class